HerPublicWebsite/Startup.cs

```diff
@@ -88,8 +89,12 @@ public void ConfigureServices(IServiceCollection services)
                 options.ModelMetadataDetailsProviders.Add(new GovUkDataBindingErrorTextProvider());
             });
 
-            // TODO: Replace this with a database based cache

```

Comment:
I think this file also has `Services.AddMemoryCache()` in it. Could you check whether that is redundant and remove it if it is please?

Reply:
It looks like the `IMemoryCache` is used to cache responses from the EPB EPC API. We _could_ distribute this cache as well, but it would require a (small) bit of work as the interfaces are different and we'd need to manually serialise/deserialise the cache entries.

I think it's probably fine to leave as-is, unless we need to be particularly efficient with the EPC API?

Reply:
Thanks for tracking that down. Fine to leave it then.